bot.py

```diff
@@ -139,15 +144,30 @@ async def get_rank(ctx, name: str):
         )
         await ctx.send(file=file, embed=create_embed(embed_data))
         # pylint: disable=broad-except
-    except Exception as e_values:
-        print(e_values)
+    except ApiError as e_values:
+        # 404 error means Data not found in API
+        if e_values.response.status_code == 404:
+            error_title = f'Summoner "{name}" is not found'
+            error_description = f"Please check the summoner name agian \n \
+                \n __*NOTE*__:   **{get_rank.name}** command only accepts one summoner name.\
+                \n\n Please type  `rank --help`  to see how to use"
+        else:
+            error_title = "Error"
+            error_description = "Oops! Something went wrong.\n\nPlease try again!"
```

Comment:
I think we can put in `Please type `rank --help` to see how to use` here as well

Reply:
Okay. I will add it